src/install.c

```diff
@@ -997,6 +997,15 @@ static gboolean handle_slot_install_plan(const RaucManifest *manifest, const RIm
 
 		/* Dummy step to indicate slot was skipped */
 		r_context_begin_step("skip_image", "Copying image skipped", 0);
+
+		/* Update the status also for skipped slots */
+		g_message("Updating slot %s status", plan->target_slot->name);
+		update_slot_status(slot_state, "ok", manifest, plan, args);
+		if (!save_slot_status(plan->target_slot, &ierror)) {
+			g_propagate_prefixed_error(error, ierror, "Error while writing status file: ");
+			return FALSE;
```

Comment:
Why don't we need the `r_context_end_step` here?

Reply:
I've checked this and would say it was just missing. The same is true for the error case of the `skip_image` step which I also fixed now.